internal/kv/kv.go

```diff
@@ -47,6 +47,7 @@ type BaseKV interface {
 	MultiRemove(keys []string) error
 	RemoveWithPrefix(key string) error
 
+	WalkWithPagination(prefix string, paginationSize int, fn func([]byte, []byte) error) error
```

Comment:
looks weird. because etcdKV is the only kv support iterate.

We could probably put this call into Meta KV interface and simply name it to walkWithPrefix(prefix, paginationSize)